src/m365/teams/commands/channel/channel-get.ts

```diff
@@ -105,9 +111,17 @@ class TeamsChannelGetCommand extends GraphCommand {
         this.teamId = teamId;
         return this.getChannelId(args);
       })
-      .then((channelId: string): Promise<Channel> => {
+      .then((channelId: any): Promise<Channel> => {
```

Comment:
`channelId` is meant to be a string. I'd suggest that we don't mix a boolean and a string here and avoid using `any`, because it will make the code hard to maintain over time. Let's keep the logic to return channel ID separate and perhaps introduce a separate method named `getChannels` that decides between using the ID or primary channel to return the requested channel.

Reply:
Reverted it back to being a string.